doc/nrf/libraries/modem/nrf_modem_lib/nrf_modem_lib_fault.rst

```diff
@@ -16,6 +16,7 @@ The behavior of the :c:func:`nrf_modem_fault_handler` function is controlled wit
 * :kconfig:option:`CONFIG_NRF_MODEM_LIB_ON_FAULT_DO_NOTHING` - This option lets the fault handler log the Modem fault and return (default).
 * :kconfig:option:`CONFIG_NRF_MODEM_LIB_ON_FAULT_RESET_MODEM` - This option lets the fault handler schedule a workqueue task to reinitialize the modem and Modem library.
 * :kconfig:option:`CONFIG_NRF_MODEM_LIB_ON_FAULT_APPLICATION_SPECIFIC` - This option lets the fault handler function :c:func:`nrf_modem_fault_handler` be defined by the application, outside of the Modem library integration layer.
+* :kconfig:option:`CONFIG_NRF_MODEM_LIB_ON_FAULT_LTE_NET_IF` - This option lets the :ref:`nrf_modem_lib_lte_net_if` handle the fault. The application can be notified about the fatal error by subscribing to the ``NET_EVENT_CONN_IF_FATAL_ERROR`` event using the :c:func:`net_mgmt_init_event_callback` function.
```

Comment:
Rather, this option is the only option available for the user to handle the fault when using `lte_net_if`. I think this should be a bit more explicit. What it does is to signal the fault with `NET_EVENT_CONN_IF_FATAL_ERROR`, which the application subscribes to using `net_mgmt_init_event_callback`.

Perhaps:
```
This is the only option available when CONFIG_NRF_MODEM_LIB_NET_IF is selected.
With this option, modem faults are signaled to the application as fatal network events (NET_EVENT_CONN_IF_FATAL_ERROR), to which the application subscribes by calling the 
net_mgmt_init_event_callback() function.
```